Gopkg.toml

```diff
@@ -41,11 +58,16 @@ required = [
   [[prune.project]]
     name = "k8s.io/code-generator"
     non-go = false
+
+  [[prune.project]]
+    name = "k8s.io/gengo"
+    non-go = false
+
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
   # branch = "master" #osdk_branch_annotation
-  version = "=v0.3.0" #osdk_version_annotation
+  version = "=v0.8.0" #osdk_version_annotation
 
 # Everything above this is required by the SDK. For each SDK version upgrade, the
 # above contents should be replaced by code copied from the SDK repo located at:
```

Comment:
This block should be updated to reflect using the operator-sdk binary to generate the dependencies.

Reply:
Ah, this is what you were talking about. Fixed in latest merge.